sql_generators/glean_usage/templates/baseline_clients_daily_v1.query.sql

```diff
@@ -24,22 +24,37 @@ WITH base AS (
     normalized_channel,
     normalized_os,
     normalized_os_version,
-    {% if has_distribution_id %}
```

Comment:
Since this is being removed here, could you also include these changes in your PR?

Reply:
Since this is the only place its used, makes sense to remove it from common.py